src/Workspaces/Core/MSBuild/Host/SimpleAnalyzerAssemblyLoaderService.cs

```diff
@@ -12,7 +12,7 @@ internal sealed class SimpleAnalyzerAssemblyLoaderService : IAnalyzerService
     {
 #if NET472
         private readonly DesktopAnalyzerAssemblyLoader _loader = new DesktopAnalyzerAssemblyLoader();
-#elif NETCOREAPP1_1 || NETCOREAPP2_1
+#elif NETCOREAPP1_1 || NETCOREAPP3_1
```

Comment:
wtf

Reply:
Yeah I think I can delete the `NETCOREAPP1_1`